src/View/Cell.php

```diff
@@ -183,9 +214,7 @@ public function render($template = null)
         };
 
         if ($cache) {
-            return $this->View->cache(function () use ($render) {
-                echo $render();
-            }, $cache);
+            return Cache::remember($cache['key'], $render, $cache['config']);
```

Comment:
Ah I see why you had to use `Cache::remember()`. `$this->View` only exists if `$render()` is invoked, which it won't be if the cache is used.